docs/tutorial/external_dependencies.rst

```diff
@@ -36,9 +36,15 @@ Then *axom* is available to be used in the DEPENDS_ON list in the following
 
 This is especially helpful for external libraries that are not built with CMake
 and don't provide CMake-friendly imported targets. Our ultimate goal is to use ``blt_register_library()`` 
-to import all external dependencies as first-class imported CMake targets to take full advanced of CMake's dependency lattice. 
+to import all external dependencies as first-class imported CMake targets to take full advantage of CMake's dependency lattice.
```

Comment:
This whole section should be reworked to use blt_import_library.  We should start discouraging the use of registered libraries except for use cases where we just cant get import libraries to work.